course/layouts/partials/course_info.html

```diff
@@ -47,16 +47,18 @@ <h4 class="course-info-title font-weight-bold">Course Info</h4>
       <tr>
         <td class="pl-0 text-nowrap">As Taught In:</td>
         <td>
-          {{ $courseData.term }}
+          {{ $courseData.term }} {{ $courseData.year }}
         </td>
       </tr>
       <tr>
         <td class="pl-0 pb-3">Level:</td>
         <td>
-          {{ $levelSearchUrl := partial "get_search_url.html" (dict "key" "level" "value" $courseData.level) }}
-          <a href="{{ $levelSearchUrl }}" class="course-info-level">
-            {{ $courseData.level }}
-          </a>
+          {{ range $courseData.level }}
```

Comment:
We should check if `$courseData.level` is an array here and only iterate it if that's the case.